### ISSUE TITLE
How to execute "slow" tests?

### ISSUE THREAD

**Issue by hannesm** [mirage/alcotest#151]
(from https://github.com/mirage/mirage-tcpip/pull/405#issuecomment-490364370)
with the current integration story, all I have to do is to execute `dune runtest` that builds and executes the tests (unless there was no modification).

but only the `` `Quick `` tests are executed by the above command. How can the `` `Slow `` tests be executed? One way is to execute the test binary itself (`_build/default/test/test.exe`), but @samoht suggested "you need to create a new test alias for this". What I don't understand is that when executing the test executable directly, it runs all the tests, but `dune runtest` only runs the `` `Quick `` ones, where is this decision/check made?

**Comment by emillon** [comment]
> where is this decision/check made?

The reason is [here](https://github.com/mirage/mirage-tcpip/blob/65e07740e6a16bdae34c2d195e1eecac308359f3/test/dune#L15): `(action)` determines how the test executable is invoked by `dune runtest`. In that case, it is passing the `-q` flag which skips slow tests.

**Comment by CraigFe** [comment]
Do we want to add some Dune-specific documentation showing an example of an Alcotest stanza?

**Comment by samoht** [comment]
yes that'd be a good idea -- even better, it would be nice if dune could support passing parameters to test stanza (currently it's not possible).
